src/components/viewSeminars/index.vue

```diff
@@ -389,16 +359,39 @@ export default {
     searchQueryVariables() {
       this.page = 1;
     },
+    SUGGESTED_SEARCH_STATE: {
+      handler(newStates, oldStates) {
+        _.forOwn(newStates, (val, key) => {
+          if (val) {
+            this.filters = _.assign(
+              _.cloneDeep(DEFAULT_FILTERS),
+              SUGGESTED_SEARCH_FILTERS[key]
+            );
+            return false;
+          } else if (oldStates[key]) {
+            const temp = _.cloneDeep(SUGGESTED_SEARCH_FILTERS[key])
+            _.forOwn(temp, (val, key) => {
+              temp[key] = !val;
+            })
+            this.filters = _.assign(_.cloneDeep(this.filters), temp);
+          }
+        })
+      }
+    },
     filters: {
       handler(newFilters) {
         // Check if new filter still matches current suggested search, if any.
-        _.entries(SUGGESTED_SEARCH_FILTERS).forEach(([key, suggested]) => {
-          suggested = _.assign(_.cloneDeep(DEFAULT_FILTERS), suggested);
+        _.forOwn(SUGGESTED_SEARCH_FILTERS, (val, key) => {
+          const suggested = _.assign(_.cloneDeep(DEFAULT_FILTERS), val);
```

Comment:
this is just refactoring, using `_.forOwn` instead of calling both `_.entries` and `.forEach `